psst-gui/src/controller/playback.rs

```diff
@@ -40,6 +41,7 @@ impl PlaybackController {
             thread: None,
             output_thread: None,
             media_controls: None,
+            volume: 50.0,
```

Comment:
I think 50 is a good default?

Reply:
I think that as the default used to be 100, we should keep it.